File: app/src/main/java/com/shalzz/attendance/ui/main/MainMvpView.java
package com.shalzz.attendance.ui.main;

import com.shalzz.attendance.billing.BillingManager;
import com.shalzz.attendance.data.model.entity.User;
import com.shalzz.attendance.ui.base.MvpView;

/**
 * @author shalzz
 */
public interface MainMvpView extends MvpView {

    BillingManager getBillingManager();

    void updateUserDetails(User user);

    void logout();
}

File: app/src/main/java/com/shalzz/attendance/ui/main/MainMvpView.kt
package com.shalzz.attendance.ui.main

import com.shalzz.attendance.billing.BillingManager
import com.shalzz.attendance.data.model.entity.User
import com.shalzz.attendance.ui.base.MvpView

/**
 * @author shalzz
 */
interface MainMvpView : MvpView {

    fun getBillingManager(): BillingManager?

    fun updateUserDetails(user: User)

    fun logout()
}
